fe/fe-core/src/main/java/com/starrocks/analysis/ShowBrokerStmt.java

```diff
@@ -31,18 +31,30 @@
 import com.starrocks.qe.ConnectContext;
 import com.starrocks.qe.ShowResultSetMetaData;
 import com.starrocks.server.GlobalStateMgr;
+import com.starrocks.sql.ast.AstVisitor;
 
 public class ShowBrokerStmt extends ShowStmt {
     public ShowBrokerStmt() {
     }
 
     @Override
-    public void analyze(Analyzer analyzer) throws AnalysisException {
-        if (!GlobalStateMgr.getCurrentState().getAuth().checkGlobalPriv(ConnectContext.get(), PrivPredicate.ADMIN)
```

Comment:
you should check Auth in PrivilegeChecker.java

Reply:
done